examples/dead-mans-switch-nft/index.rsh

```diff
@@ -0,0 +1,74 @@
+'reach 0.1';
+'use strict';
+
+export const main = Reach.App(() => {
+  const Creator = Participant('Creator', {
+    transferTo: Fun([], Address),
+  });
+  const Owner = ParticipantClass('Owner', {
+    notifyReceive: Fun([], Null),
+    notifyRenewNeeded: Fun([], Null),
+    notifyRenewSuccess: Fun([], Null),
+    notifyRenewFail: Fun([], Null),
+  });
+  const vNFT = View('NFT', { owner: Address });
+
+  init();
+
+  Creator.publish();
+
+  // const ownerOnly = (currentOwner, fn) =>
+  //   Owner.only(() => { if (this == currentOwner) { fn(interact); } });
+
+  var owner = Creator;
```

Comment:
First, I think you should use `parallelReduce`

Second, the idea is that there's always an "owner" and an "heir". If the owner doesn't ping, then the heir can claim it and become the new owner (setting up the next heir).